test/object_store/conftest.py

```diff
@@ -14,6 +14,8 @@
 
 
 def pytest_addoption(parser):
+    parser.addoption('--keep-tmp', action='store_true',
```

Comment:
The test.py has `-s` key that preserves logs and tempdirs either. It looks like this new `--keep-tmp` should be set by test.py's `-s`, WDYT?

Reply:
i don't use `test.py` unless i have to. `--keep-tmp` is only used for debugging purpose when something really strange happens, it is not supposed to be use in our daily development.

Reply:
i took a look at how `--save-log-on-success` work, in case you are interested in this feature. i turns out, it assume that the scylla cluster (instance) is created by `test.py`, instead of by the test itself, so it is able to collect what is left by the scylla instance instead of removing them. 

in our case, the scylla instance is managed by ourselves, and `test.py` does not offer a mechinary passing an argument to a certain test.  please see `PythonTest._prepare_pytest_params()` .  we could create such a machinery for this very test. but the question is: is it worthy to do this?

Reply:
I use test.py and I do remember that I was missing logs from object_store test sometimes. But since it seems there's no easy way to pass the `-s` down to `--keep-tmp`, just restarting the object_store test with `--keep-tmp` would work for me. Thanks!

Reply:
which log are you missing? the one from scylla or the one from minio server?

Reply:
Sorry. Not logs, but the minio working directory altogether to look for sstables objects it puts. The log was preserved by `_remove_all_but(tempdir, 'log')` IIRC

Reply:
@xemul okay, if it is minio's log, https://github.com/scylladb/scylladb/pull/14513 to the rescue! actually, i also plan to add an option to keep the minio server around to i can peek at it after the test finishes. as i want to ensure the sstables are uploaded. or i will have to nuke the sstables in local fs, and restart the scylla instance to see if the scylla can be boot from s3 or not.

Reply:
probably the minio server's log does not help a lot. as it is missing the access log, like a regular http server has...

Reply:
ahh, right.  minio server helper script also prints out the necessary `export` commands when it launches minio server, so it's relatively straightforward to feed info of the external minio to the test manually.